webapi/appsettings.json

```diff
@@ -161,8 +161,8 @@
   "DocumentMemory": {
     "GlobalDocumentCollectionName": "global-documents",
     "ChatDocumentCollectionNamePrefix": "chat-documents-",
-    "DocumentLineSplitMaxTokens": 30,
-    "DocumentParagraphSplitMaxLines": 100,
+    "DocumentLineSplitMaxTokens": 64,
```

Comment:
We actually have defaults and documented why we selected these numbers here: https://github.com/microsoft/chat-copilot/blob/main/webapi/Options/DocumentMemoryOptions.cs#L32. Need to adjust the comments on why we selected the new numbers.

Reply:
Above this Json block, we also have a link to the OpenAI explaining why we selected the token limits for chunking. Will need to rewrite that too.

Reply:
Thank you for this call-out.  Consistency is key, esp from customer perspective.

Reply:
Aside: (I've read the material)  Part of the issue is that we've conflated our chunk boundary with _paragraph_.  A paragraph may very well be average 100 tokens, but this doesn't necessarily equivalent to document chunking granularity...esp for structured documents data.  In short, I don't sree that the [Open AI article](https://help.openai.com/en/articles/4936856-what-are-tokens-and-how-to-count-them) isn't directly recommending document chunking strategy.

We could rename the setting from `DocumentParagraphSplitMaxLines  `to `DocumentSplitMaxLines` ...also the parameter is actually `maxTokensPerParagraph`...so actually maybe  `DocumentSplitMaxTokens` ?

Reply:
Updated